### PR TITLE
Tng 794

### PR DESCRIPTION
**Pull request by rrieger-espirit**
In this context, tests for the LineSeparator was created, also some bugfixing in tests

### CODE REVIEW

File: src/components/ProductListItem/index.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ class ProductListItem extends BaseComponent<ProductListItemProps> {
           class="w-full h-full m-0 border-0"
           src={this.image.src}
           resolutions={this.image.resolutions}
-          lazy={true}
```
Comment (vlapl) [inline src/components/ProductListItem/index.tsx:29]: Das ist, um den Test zu fixen? Ich habe dazu in meinem PR einen anderen Fix, mit dem man nicht auf "lazy" verzichten braucht: https://github.com/e-Spirit/fsxa-ui/pull/54/commits/801590b073693d5ae25259ffeb25cb9ace702a07

File: src/components/LineSeparator/__tests__/LineSeperator.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { render } from "@testing-library/vue";
+import LineSeperator from "./../";
+
+describe("components/LineSeperator", () => {
+  it("renders the correct default", () => {
+    const { getByTestId } = render(LineSeperator, {});
```
Comment (dbarczewski) [inline src/components/LineSeparator/__tests__/LineSeperator.spec.tsx:6]: Please remove unnecessary empty object.

File: src/components/LineSeparator/__tests__/LineSeperator.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        side,
+      },
+    });
+    const separator = getByTestId("separator-test");
+    const flex = getByTestId("flex-test");
+    expect(separator.classList.contains("h-1")).toBeTruthy();
+    expect(flex.classList.contains("flex-row-reverse")).toBeTruthy();
+    expect(separator.classList.contains("w-8")).toBeTruthy();
+  });
+  it("renders responsive", () => {
```
Comment (dbarczewski) [inline src/components/LineSeparator/__tests__/LineSeperator.spec.tsx:28]: Please provide a better test message.

File: src/components/LineSeparator/__tests__/LineSeperator.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
+import { render } from "@testing-library/vue";
```
Comment (dbarczewski) [inline src/components/LineSeparator/__tests__/LineSeperator.spec.tsx:1]: Please rename the file to "LineSeparator.spec.ts"

**Review by dbarczewski — CHANGES_REQUESTED** [review]
Great job so far. You only need to make a few small improvements.

File: src/components/LineSeparator/__tests__/LineSeperator.spec.ts [hunk 5]
```diff
@@ -1,9 +1,9 @@
 import { render } from "@testing-library/vue";
-import LineSeperator from "./../";
+import LineSeperator from "..";
```
Comment (dbarczewski) [inline src/components/LineSeparator/__tests__/LineSeperator.spec.ts:2]: Please check the spelling for the import and the file name.

Comment (dbarczewski) [reply src/components/LineSeparator/__tests__/LineSeperator.spec.ts:2]: And import the component via `./../`

**Review by dbarczewski — CHANGES_REQUESTED** [review]

**Review by dbarczewski — APPROVED** [review]
Nice, well done.

**Pull request merged**